recipes/lytekit/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set name = "lytekit" %}
+{% set version = "0.15.27" %}
+{% set python_max = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/lytekit-{{ version }}.tar.gz
+  sha256: bf3d9e127ad02b94e88c9b7f4f8df2e81946104c470c935d00ad99b312f46915
+
+build:
+  skip: true  # [py<37 or py>310]
+  entry_points:
+    - pyflyte-execute=flytekit.bin.entrypoint:execute_task_cmd
+    - pyflyte-fast-execute=flytekit.bin.entrypoint:fast_execute_task_cmd
+    - pyflyte-map-execute=flytekit.bin.entrypoint:map_execute_task_cmd
+    - pyflyte=flytekit.clis.sdk_in_container.pyflyte:main
+    - flyte-cli=flytekit.clis.flyte_cli.main:_flyte_cli
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - lyteidl ==0.2.1
+    - wheel >=0.30.0,<1.0.0
+    - click >=6.6,<9.0
+    - croniter >=0.3.20,<4.0.0
+    - deprecated >=1.0,<2.0
+    - docker-py >=7.1.0
+    - python-dateutil >=2.1
+    - grpcio >=1.43.0,!=1.45.0,<2.0
+    - grpcio-status >=1.43,!=1.45.0
+    - protobuf >=3.6.1,<4
+    - python-json-logger >=2.0.0
+    - pytimeparse >=1.1.8,<2.0.0
+    - pytz
+    - keyring >=18.0.1
+    - requests >=2.18.4,<3.0.0
+    - responses >=0.10.7
+    - sortedcontainers >=1.5.9,<3.0.0
+    - statsd >=3.0.0,<4.0.0
+    - urllib3 >=1.22,<2.0.0
+    - wrapt >=1.0.0,<2.0.0
+    - retry ==0.9.2
+    - dataclasses-json >=0.5.2,<=0.5.6
+    - jsonschema >=4.5.1
+    - marshmallow-jsonschema >=0.12.0
+    - marshmallow-enum
+    - natsort >=7.0.1
+    - docker-image-py >=0.1.10
+    - singledispatchmethod  # [py<38]
```

Here's the selector that I found the linter was complaining about!
```suggestion
    - singledispatchmethod
```

---

The current `python_min` is 3.9, so this dependency could be altogether removed.